src/lib_json/json_reader.cpp

```diff
@@ -193,19 +193,43 @@ bool Reader::readValue() {
     currentValue().swapPayload(v);
     currentValue().setOffsetStart(token.start_ - begin_);
     currentValue().setOffsetLimit(token.end_ - begin_);
-  } break;
+  } 
+  break;
   case tokenFalse: {
     Value v(false);
     currentValue().swapPayload(v);
     currentValue().setOffsetStart(token.start_ - begin_);
     currentValue().setOffsetLimit(token.end_ - begin_);
-  } break;
+  } 
+  break;
   case tokenNull: {
     Value v;
     currentValue().swapPayload(v);
     currentValue().setOffsetStart(token.start_ - begin_);
     currentValue().setOffsetLimit(token.end_ - begin_);
-  } break;
```

Comment:
Nit: You should run clang format on this patch. We prefer `break` on the same like as the closing bracket of a scoped switch statement, so this line should be reverted and the following lines updated. 

If you haven't used clang format before, I added a section to our CONTRIBUTING.md documentation, and you can use the following command, assuming you are using lib type shared:

```
ninja -C build-shared clang-format
```

Reply:
If you run clang-format on this patch, we can work on getting it checked in.